examples/test/local/test_bert_cuda_gpu.py

```diff
@@ -6,13 +6,14 @@
 from pathlib import Path
 
 import pytest
-from utils import check_output, patch_config
+
+from ..utils import check_output, patch_config
 
 
 @pytest.fixture(scope="module", autouse=True)
 def setup():
     """Setups any state specific to the execution of the given module."""
-    cur_dir = Path(__file__).resolve().parent.parent
+    cur_dir = Path(__file__).resolve().parent.parent.parent
```

Comment:
not directly related to this PR but perhaps we can make this a util like `get_cur_dir`

Reply:
The test still needs `cur_dir` 3 lines after (or maybe not. I can verify this)

Reply:
oh I meant, we can make this line 16 use a utils function since almost all examples do the same thing and we are hardcoding the directory structure in each one. I just called it `get_examples_dir` since it is actually returning the `olive/examples` directory path and not the current directory (cur_dir is an old name from before example test refactor)

Reply:
I just refactored all of this.